docs/installation.md

```diff
@@ -217,8 +217,8 @@ services:
   # Uncomment the following lines if you are using the `hybrid/latest` or `redis` image.
   # redis:
   #   image: redis:latest
-  #   ports:
-  #     - 6379:6379
+ 
+  
```

Comment:
```suggestion
```

Thanks :heart: for working on such a wonderful PR. We really appreciate it :+1: Keep it up!! But I would like to suggest a minor change, can you please remove those two empty new lines because I think they look very unnecessary in the code snippet :slightly_smiling_face: . What do you think?